config.yaml

```diff
@@ -5,3 +5,7 @@ options:
       Spring Boot configuration properties in JSON format, it will be passed to 
       the Spring Boot application as SPRING_APPLICATION_JSON environment variable.
     default: ""
+  jvm-config:
+    type: string
+    description: JVM configuration that will be passed to JAVA_TOOL_OPTIONS environment variable.
```

Comment:
s/that will be passed to/to be passed as

Reply:
Updated, thanks!